media-gfx/nvidia-texture-tools/nvidia-texture-tools-2.0.8-r4.ebuild

```diff
@@ -44,7 +45,7 @@ S="${WORKDIR}/${PN}"
 
 src_prepare() {
 	edos2unix cmake/*
-	EPATCH_SUFFIX=patch epatch "${WORKDIR}/patches"
+	EPATCH_SUFFIX=patch eapply "${WORKDIR}/patches"
```

Comment:
tried adding `"${WORKDIR}/patches"` to `PATCHES` as last line?

Reply:
that may work, actually...

Reply:
Does work, nice. Will have to remember this in the future.